javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/DescriptorTest.java

```diff
@@ -20,6 +21,7 @@
     void descriptorTest() {
         String code = 
                 "public class A {\n" +
+                "  A(int i, double d, Thread t) {return new Object();}\n" +
```

Comment:
Even if the test is correct/pass, this is not a correct implementation for a constructor. A constructor cannot return a value. Can you change it please?

Reply:
Good catch. I've removed the return statement.